addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -40,12 +29,25 @@ def _default_deposit_account_id(self):
     def _default_deposit_taxes_id(self):
         return self._default_product_id().taxes_id
 
+    @api.model
+    def _default_has_down_payment(self):
+        if self._context.get('active_model') == 'sale.order' and self._context.get('active_id', False):
+            sale_order = self.env['sale.order'].browse(self._context.get('active_id'))
+            return sale_order.order_line.filtered(
+                lambda sale_order_line: sale_order_line.is_downpayment
+            )
+
+        return False
+
     advance_payment_method = fields.Selection([
-        ('delivered', 'Invoiceable lines'),
-        ('all', 'Invoiceable lines (deduct down payments)'),
+        ('delivered', 'Standard invoice'),
         ('percentage', 'Down payment (percentage)'),
         ('fixed', 'Down payment (fixed amount)')
-        ], string='What do you want to invoice?', default=_get_advance_payment_method, required=True)
+        ], string='What do you want to invoice?', default='delivered', required=True,
+        help="A standard invoice is issued with all the order lines ready for invoicing, \
+        according to their invoicing policy (based on ordered quantity or on delivered quantity).")
```

Comment:
make it shorter: based on orderded or delivered quantity.

Reply:
@fgi-odoo text updated